eoscostarica/proton-affiliate
# feat(webapp): use env variable for contract account

### What does this PR do?

- Resolve #88

### Steps to test

1. make run
2. login as an admin
3. check the referral table

#### CheckList
